File: algorithm10/algorithm10_1_2.c
#include <stdio.h>
#include <string.h>

int isOdd(int n){
    // AND -> 홀수이면 1, 짝수이면 0
    return(n&1);
}


int main() {
    int n;
    scanf("%d", &n);
    if(isOdd(n))
        printf("Odd");
    else
        printf("Even");
    
    return 0;
}

File: algorithm10/algorithm10_1_2.java
import java.io.*;
import java.util.*;

class Main {
    static boolean isEven(int n) {
        return ((n & 1)!=1);
    } 
    
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        
        System.out.print(isEven(n) == true ? "Even" : "Odd");
        
	}
}